packages/ui/src/components/Renderer.tsx

```diff
@@ -94,7 +98,7 @@ Check this document: https://pdfme.com/docs/custom-schemas`);
         ref.current.innerHTML = '';
       }
     };
-  }, [value, JSON.stringify(schema), JSON.stringify(options), mode, scale]);
+}, [mode, scale, schemaRerenderState, JSON.stringify(options)]);
```

Comment:
`value == schema.content` so changes would be caught anyway.

Reply:
hmm this change doesn't work now for removing the image file on an image schema. This needs a re-think.